### PR TITLE
Add rubygem-aws-sigv4 package

### PR DESCRIPTION
**Pull request by timogoebel**
Please build for:

* [x] Nightly
* [x] 1.21
* [x] 1.20

This is a dependency for #3532

### CODE REVIEW

File: packages/plugins/rubygem-aws-sigv4/rubygem-aws-sigv4.spec [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+Requires: %{?scl_prefix}rubygem(aws-eventstream) >= 1.0.2
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+# end specfile generated dependencies
+
+%description
+Amazon Web Services Signature Version 4 signing ligrary. Generates sigv4
```
Comment (ekohl) [inline packages/plugins/rubygem-aws-sigv4/rubygem-aws-sigv4.spec]:
```suggestion
Amazon Web Services Signature Version 4 signing library. Generates sigv4
```

File: packages/plugins/rubygem-aws-sigv4/rubygem-aws-sigv4.spec [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+%global gem_name aws-sigv4
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 1.1.0
+Release: 1%{?dist}
+Summary: AWS Signature Version 4 library
+Group: Development/Languages
+License: Apache-2.0
+URL: http://github.com/aws/aws-sdk-ruby
```
Comment (ekohl) [inline packages/plugins/rubygem-aws-sigv4/rubygem-aws-sigv4.spec]:
```suggestion
URL: https://github.com/aws/aws-sdk-ruby
```

**Comment by ekohl** [comment]
Could you rebase for the merge conflicts?

File: packages/plugins/rubygem-aws-sigv4/rubygem-aws-sigv4.spec [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name aws-sigv4
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 1.1.0
+Release: 1%{?dist}
+Summary: AWS Signature Version 4 library
+Group: Development/Languages
+License: Apache-2.0
```
Comment (ekohl) [inline packages/plugins/rubygem-aws-sigv4/rubygem-aws-sigv4.spec]:
```suggestion
License: ASL 2.0
```

**Comment by timogoebel** [comment]
Rebased and addressed the comments.

**Comment by timogoebel** [comment]
> It's odd that it's reordering the comps files. Locally I don't get that. Perhaps it's a locale difference? Could you regenerate the comps files by running `./comps_doc.sh`?

I reverted the changed and did it manually. Might just be my Mac.

**Comment by ekohl** [comment]
It does rely on the en_GB locale. Perhaps that's missing which can result in a different ordering.

**Pull request merged**